R/FilterWrapper.R

```diff
@@ -11,12 +11,16 @@
 #'   filters. See [listFilterMethods]. Can only be used in combination with
 #'   ensemble filters. See [listFilterEnsembleMethods].
 #' @param fw.perc (`numeric(1)`)\cr If set, select `fw.perc`*100 top scoring
-#'   features. Mutually exclusive with arguments `fw.abs` and `fw.threshold`.
+#'   features. Mutually exclusive with arguments `fw.abs`, `fw.threshold` and `fw.func.
 #' @param fw.abs (`numeric(1)`)\cr If set, select `fw.abs` top scoring features.
-#'   Mutually exclusive with arguments `fw.perc` and `fw.threshold`.
+#'   Mutually exclusive with arguments `fw.perc`, `fw.threshold` and `fw.func`.
 #' @param fw.threshold (`numeric(1)`)\cr If set, select features whose score
-#'   exceeds `fw.threshold`. Mutually exclusive with arguments `fw.perc` and
-#'   `fw.abs`.
+#'   exceeds `fw.threshold`. Mutually exclusive with arguments `fw.perc`, `fw.abs`
+#'   and `fw.func`.
+#' @param fw.func (`function)`)\cr . If set, select features via a custom thresholding 
+#'   function, which must return the number of top scoring features to select\cr
+#'   Mutually exclusive with arguments `fw.perc`, `fw.abs` and `fw.threshold`.
+#' @param func.args (any)\cr Arguments passed to the custom thresholding function
```

Comment:
Could you add an example of how this can be used to the documentation please?

Reply:
Yes I have done that.